### PR TITLE
Add actions-reducer

### PR DESCRIPTION
**Pull request by megha-n-bodke**
- Use ducks-pattern.
- Define and export action types for adding and removing a book.
- Define an action type for checking the status.
- Write reducer and export it as default.
- Configure the Redux Store

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+export const addBook = () => ({
+  type: 'ADD_BOOK',
+});
+
+export const removeBook = () => ({
+  type: 'REMOVE_BOOK',
+});
```
Comment (phelian23) [inline src/redux/books/books.js]: - [ ] Kindly define the action types `ADD_BOOK` and `REMOVE_BOOK` as variables and it should follow the duck pattern and then use it in your types as requested in the project requirements. for instance
``` javascript
  const ADD_BOOK = books/books/ADD_BOOK
```

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const categories = [];
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (phelian23) [inline src/redux/categories/categories.js]: - [ ] Kindly make use of the duck pattern to define the action type check_status. e.g
``` javascript
  const CHECK_STATUS = bookstore/categories/CHECK_STATUS
```

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @megha-n-bodke,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
No linter errors ✔️
Repo looks professional 💼
Solid pull request 📜

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @megha-n-bodke , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**